### ISSUE TITLE
Requesting Kicad files for rev 2.0WHS

### ISSUE THREAD

**Issue by Bombergames** [mattdibi/redox-keyboard#114]
It looks like the rev 1 kicad files were updated with the 2.0 commit, but it doesn't have the actual updated files for the rev 2.0 board.

**Comment by jesper1266** [comment]
Hi, I second that :-)
I would like to help.
It would for example be nice tha the pinheader for uploading the firmware is the same for both the board and reciever.

Cheers
Jesper

**Comment by mattdibi** [comment]
Hi guys,

unfortunately I'm having issues with recovering the files: my Windows installation decided to brick itself after an update and cannot access it anymore. I'll try and upload the files when/if I recover them.

**Comment by PhiBabin** [comment]
Hi guys,
I rebuilded the KiCad of rev 2 by using the rev 1 as a starting point and using the gerber files of the rev 2 as reference. I included some of the modified footprints its own library.

KiCad project file: [redox_rev2_recovery_kicad.zip](https://github.com/mattdibi/redox-keyboard/files/7575625/redox_rev2_recovery_kicad.zip)
Gerber file from rebuild: [redox_rev2_recovery.zip](https://github.com/mattdibi/redox-keyboard/files/7575623/redox_rev2_recovery.zip)

Original gerber top layer:

![top-original](https://user-images.githubusercontent.com/83554033/142746161-c85b5198-8fe1-40bf-ab08-1ed98183d2b0.png)

Rebuild gerber top layer:
![top](https://user-images.githubusercontent.com/83554033/142746202-9f5835b3-ec05-4e13-bc73-b69cd764ad4c.png)

Original gerber bot layer:
![bottom-original](https://user-images.githubusercontent.com/83554033/142746177-c1b791a0-7790-4aa9-a647-b2fce3416312.png)

Rebuild gerber bot layer:
![bottom](https://user-images.githubusercontent.com/83554033/142746206-efb6bfe2-5bd9-4b8b-811d-c29f7457c3ed.png)

This is my first time using KiCad, so use this at your risk and peril.

**Comment by KeyTamer** [comment]
First thanks, a lot for the KiCad file! 
A other think. Did you test it all ready?

**Comment by PhiBabin** [comment]
Yeah, I send it to AllPCB and they worked fine. I'm typing on it right now.
The only issues I had are : 
 - The diode direction is not visible because the pads are too close to the marking. However, from looking at the image of the original, it seems that it has the same issue. This doesn't really matter, since the guide said that you just have to oriented the diode toward the square pad. It should be super to fix that in a future version.
 - The holes for the hotswap are not circular, they are skew (see last picture). This mean that when you insert each hotswap you get a very satisfying *click*. It also mean that your fingers are going to hurt after putting 70 hotswaps. I don't think the kidcad project is the cause of this, this is probably an issue with how I exported the Gerber or with AllPCB or a normal issue when doing overlapping hole like this. Anyway, it really doesn't matter and it's a very minor issue.

Note : The version that I printed is different from the KiCad project files shared above, I added the option of putting a rotary encoder at the location used by key K16. I want to finish the QMK+firmware changes before sharing it.
![20211202_183706](https://user-images.githubusercontent.com/867925/146862887-d127c8e1-4fec-4968-81e5-ed66d71ff73e.jpg)
![20211202_183711](https://user-images.githubusercontent.com/867925/146862893-e6234053-b13b-4842-bc97-f0c3aaf99658.jpg)
![20211203_191945](https://user-images.githubusercontent.com/867925/146863331-19cdbfdb-e80c-41bd-96b6-48ee7b0dbde3.jpg)

**Comment by widget-** [comment]
@PhiBabin I'm interested in trying this out and probably opening a pull request into this repo assuming it goes well. Can you confirm that you're releasing it under the same [MIT license](https://github.com/mattdibi/redox-keyboard/blob/master/LICENSE) as the original project?

**Comment by PhiBabin** [comment]
Yes, I'm  releasing it under MIT license.
Sorry forgot to upload them a fork.

**Comment by l4u** [comment]
@PhiBabin Do you have a forked version or a pull request please? I'd like to make modifications based on your version with proper git history.

**Comment by PhiBabin** [comment]
@l4u  sorry for the delays... https://github.com/mattdibi/redox-keyboard/pull/151
